Assets/MixedRealityToolkit/Utilities/Physics/ManipulationMoveLogic.cs

```diff
@@ -8,15 +8,15 @@
 namespace Microsoft.MixedReality.Toolkit.Physics
 {
     /// <summary>
-    /// Implements a movement logic that uses the model of angular rotations along a sphere whose 
-    /// radius varies. The angle to move by is computed by looking at how much the hand changes
-    /// relative to a pivot point (slightly below and in front of the head).
+    /// Implements a move logic that will move an object based on the initial position of 
+    /// the grab point relative to the pointer and relative to the object, and subsequent
+    /// changes to the pointer and the object's rotation
     /// 
     /// Usage:
     /// When a manipulation starts, call Setup.
     /// Call Update any time to update the move logic and get a new rotation for the object.
     /// </summary>
-    public class TwoHandMoveLogic
+    public class ManipulationMoveLogic
```

Comment:
@wiwei should we move this, `TwoHandRotateLogic` and `TwoHandScaleLogic` to some internal namespace/location? They're not intended for users for normal usage and I've broken the public interface to these classes twice now. It would be nice if we could make it clear that these classes are subject to change.

Reply:
can't we just define those classes internal instead of public?

Reply:
there's no reason imho to have those public

Reply:
+1 on having these just be internal as well.

This is a breaking change in that we're removing the visibility of this, but this is a case where it should be a fairly obvious implementation detail of the the manipulation handler.

---
In reply to: [324756550](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5955#discussion_r324756550) [](ancestors = 324756550)